packages/riverpod_generator/README.md

```diff
@@ -169,12 +169,29 @@ targets:
           # Could be changed to "Pod", such that riverpod_generator
           # would generate "countPod" instead of "countProvider"
           provider_name_suffix: "Provider" # (default)
-          # Similar to provider_name_sufix, this is an option for renaming
+          # Similar to provider_name_suffix, this is an option for renaming
           # providers with parameters ("families").
           # This takes precedence over provider_name_suffix.
           provider_family_name_suffix: "Provider" # (default)
 ```
 
+You can change provider name prefix in the build.yaml file:
+
+```yaml
```

Comment:
I don't think this warrants a separate code snippet. I'd fuse the doc for prefixes and suffixes

Reply:
I fused them in [doc: Fuse the doc for prefixes and suffixes](https://github.com/rrousselGit/riverpod/pull/3348/commits/88fb9d719faa84d435eceb7d83bc53dd716ac7ce)